docs/director/03-tenancy.md

```diff
@@ -0,0 +1,40 @@
+# Tenancy
+Tenant is a single object that owns resources. Every tenant's resources are isolated from the other tenants, which ensures data security and privacy.
+Compass Director is a multi-tenant service, which means that a single instance of Director is shared among different tenants (clients), but their data is isolated.
```

Comment:
Let's discuss whether to leave this sentence here or move it to the next paragraph.